### ISSUE TITLE
Option to Remove Drag Space

### ISSUE THREAD

**Issue by loganmajor** [LorenzCK/OnTopReplica#72]
I see an annoying white bar at the top of the thumbnailed window. I presume this is intended to be "drag space" so the cursor can easily move the window around. However, this feature is bothersome to me.

Please add a toggle for the drag space! Thanks in advance.

Picture reference: https://vgy.me/BBkfcA.png

**Comment by LorenzCK** [comment]
This has been annoying me as well for some time: it's due to a bug (or a new weird behavior) of title-less windows on Windows&nbsp;10. I have been looking for a workaround for some time and there appears to be no way to avoid it completely: the next version of OnTopReplica (which will finally support Windows 10) will probably have a full "standard" title bar (which can then be hidden together with the window borders if you click on the “show chrome” option).

**Comment by blundercode** [comment]
Still an issue, any workarounds for the moment?

**Comment by loganmajor** [comment]
> Still an issue, any workarounds for the moment?

" I have been looking for a workaround for some time and there appears to be no way to avoid it completely," said LorenzCK on 2018-Dec-17.

It will be an issue indefinitely with no workarounds yet.

**Comment by blundercode** [comment]
Yeah, but it has been like 8 months of changes in windows. I bet the creator of https://winaero.com/comment.php?comment.news.1836 knows how to solve it, I used to use this to tweak a lot of windows stuff alongside OnTopReplica so I bet there's a way. I'll work on it.

**Comment by loganmajor** [comment]
> Yeah, but it has been like 8 months of changes in windows. I bet the creator of https://winaero.com/comment.php?comment.news.1836 knows how to solve it, I used to use this to tweak a lot of windows stuff alongside OnTopReplica so I bet there's a way. I'll work on it.

If you know a solution to the problem, OnTopReplica is open source and you (or someone you know) can modify it to your hearts content.
